src/main/java/org/gitlab4j/api/models/HealthCheckStatus.java

```diff
@@ -0,0 +1,17 @@
+package org.gitlab4j.api.models;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+public class HealthCheckStatus {
+    private String status;
```

Comment:
Ideally, these should be enums but I don't know all the possible values other than "ok". If you have any ideas, I would be happy to update this.